build.cake

```diff
@@ -0,0 +1,150 @@
+//////////////////////////////////////////////////////////////////////
+// TOOLS
+//////////////////////////////////////////////////////////////////////
+
+#tool "nuget:?package=GitVersion.CommandLine&version=3.6.5"
+
+//////////////////////////////////////////////////////////////////////
+// ARGUMENTS
+//////////////////////////////////////////////////////////////////////
+
+var target = Argument("target", "Default");
+var configuration = Argument("configuration", "Release");
+var isCIBuild = !BuildSystem.IsLocalBuild;
+
+var projects = "./**/*.csproj";
+var testProjects = "./test/**/*.csproj";
```

Comment:
Wouldn't projects collection also include testProjects because of path pattern? I suggest to remove everything related to tests because this repo doesn't have them.

Reply:
It's true we don't have tests in the repo **just yet**, though my idea was to keep changes to the original `build.cake` from `Checkout.Heartbeat` to minimum - so that our build scripts across all our Nuget packages would be unified

Reply:
So, there are two problems: 
1.  `var projects = "./**/*.csproj";` contains all projects, including test projects. Then `allProjectFiles`  collection contains duplicate projects
2. Looks strange, but ok, we can keep test-related logic. Then non-test projects should be moved to `src` folder otherwise this script is confusing

Reply:
Agree, we should probably move the code to `src` folder - to align with the rest of our projects...